datamade/bga-payroll
# Remove calculated salary header from pre-processed data

## Description

See title. This change is necessary because we are no longer calculating total pay, so adding the salary column will result in processed data files with invalid dimensions (one more header field than data fields in subsequent rows). This PR also removes an unexpected/unnecessary extra column from the raw 2017, part 3 file.

## Testing instructions

- Remove any generated data: `make clean`
- Remake the data: `docker-compose run --rm app make`
- Verify each generated file has equal header and data rows by running the following command for each file: `head FILENAME | (docker-compose run --rm app csvlook --no-inference) | less`, where `FILENAME` is `payroll-actual-2017-pt-1.csv`, etc. Confirm that you don't see errors about invalid dimensions, and that the headers line up with the data.
